pyEpiabm/pyEpiabm/tests/test_initial_infected_sweep.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+import pyEpiabm as pe
+
+
+class TestInitialInfectedSweep(unittest.TestCase):
+    """Test the 'InitialInfectedSweep' class.
+    """
+    @classmethod
+    def setUpClass(cls) -> None:
+        """Sets up a population we can use throughout the test.
+        2 people are located in one microcell.
+        """
+        cls.pop_factory = pe.ToyPopulationFactory()
+        cls.test_population = cls.pop_factory.make_pop(2, 1, 1, 1)
+        cls.cell = cls.test_population.cells[0]
+        cls.person1 = cls.test_population.cells[0].microcells[0].persons[0]
+        cls.person2 = cls.test_population.cells[0].microcells[0].persons[1]
+
+    def test_call(self):
+        """Test the main function of the Initial Infected Sweep.
+        """
+        test_sweep = pe.InitialInfectedSweep()
+        test_sweep.bind_population(self.test_population)
+        # Test asking for more infected people than the population number
+        # raises an error.
+        params = {"population_size": 2, "initial_infected_number": 4}
```

Comment:
```suggestion
        params = {"initial_infected_number": 4}
```
Is there any reason to include population_size here and below, since InitialInfectedSweep currently only uses infected_number

Reply:
Right, I corrected it in a commit.